### ISSUE TITLE
Migration to Resonite Ready

### ISSUE THREAD

**Issue by stiefeljackal** [Earthmark/Neos-TypeType#4]
Hello! It just wanted to let you know that I forked the repo and updated the mod to work with Resonite. All DLLs have been updated to point to the new Resonite DLLs; code and unit tests were also updated as a result. When the modding group makes the decision on the official loader, I will make sure to do one final update and give those changes back to you via a PR or through other means that you prefer.

Fork: https://github.com/stiefeljackal/Resonite-TypeType

**Comment by Earthmark** [comment]
Sweet, thank you for making the migration! I wasn't sure if people actually used this mod.

I skimmed the changes and they looked pretty valid, but I can take an actual review pass later tonight or tomorrow.

If you want to, please make a pr with me marked as a reviewer, that way I can more easily see it in my queue.

**Comment by stiefeljackal** [comment]
Created PR #5 and marked as ready for your review.

**Comment by stiefeljackal** [comment]
Delta released the official mod loader. I’ll make the final PR today to update any necessary information. Once that is done, this mod will be ready for the full Resonite mod experience. :)

**Comment by Earthmark** [comment]
I pushed a new release and updated the config in this repo, it just needs to get merged into the mod manifest repo.

**Comment by stiefeljackal** [comment]
I see that, but the README needs to be updated to point to [Delta's Official Mod Loader](https://github.com/resonite-modding-group/ResoniteModLoader) and the disclaimer above needs to be removed. The README is the only file that needs to be updated so you can do that if you want. The DLL might need to be recompiled to Delta's Mod Loader DLL as well

**Comment by Earthmark** [comment]
Good call, I'll do an update later today or tomorrow
